[2023-11-25 01:56:00,256] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:01,152] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:01,169] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:06,903] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:06,903] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:06,903] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:07,736] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:07,736] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:13,044] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 01:56:13] "OPTIONS /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:56:13,070] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 01:56:13] "[35m[1mGET /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1[0m" 500 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:51,386] INFO _log() L224   * Detected change in 'C:\\venv\\.__Projetos\\PUC_EngSoft_MVP4\\back\\AvalFIMult.py', reloading - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:51,534] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:52,295] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:52,299] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:57,443] INFO _log() L224  [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5001 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:57,445] INFO _log() L224  [33mPress CTRL+C to quit[0m - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:57,449] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:58,531] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:58:58,531] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:59:02,396] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 01:59:02] "OPTIONS /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:59:03,547] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 01:59:03] "OPTIONS /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:59:13,653] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 01:59:13] "[35m[1mGET /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1[0m" 500 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 01:59:13,673] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 01:59:13] "[35m[1mGET /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1[0m" 500 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:03:21,984] INFO _log() L224   * Detected change in 'C:\\venv\\.__Projetos\\PUC_EngSoft_MVP4\\back\\AvalFIMult.py', reloading - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:03:22,348] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:03:23,284] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:03:23,290] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:03:27,547] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 02:03:27] "OPTIONS /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1" 200 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:03:28,213] INFO _log() L224  127.0.0.1 - - [25/Nov/2023 02:03:28] "[35m[1mGET /AvalFIMult?resgate=1&capta=2&cotistas=1000&patliq=1000000&quota=5 HTTP/1.1[0m" 500 - - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:07:46,002] INFO _log() L224   * Detected change in 'C:\\venv\\.__Projetos\\PUC_EngSoft_MVP4\\back\\AvalFIMult.py', reloading - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:07:46,329] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:07:47,235] WARNING _log() L224   * Debugger is active! - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
[2023-11-25 02:07:47,238] INFO _log() L224   * Debugger PIN: 738-487-147 - call_trace=C:\Users\moacy\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\werkzeug\_internal.py L224 
